File: Blatt1/Blatt01-Beispielcode/Blatt01-Beispielcode/scala/primtest.scala
import scala.io.Source
import scala.util.{Try, Success, Failure}

def main(args: Array[String]): Unit = {
  var number = 10007*100003
  //Test if number is prime
  var i = 2
  var isPrime = true
  while (i <= math.floor(math.sqrt(result))) {
    if (result % i == 0) {
      isPrime = false
    }
    i += 1
  }
  if (isPrime) {
    println(s"$result is a prime number")
  } else {
    println(s"$result is not a prime number")
  }
}

File: Blatt1/Blatt01-Beispielcode/Blatt01-Beispielcode/java/primtest.java
import java.io.File;
import java.io.FileNotFoundException;
import java.util.Scanner;

public class primtest {
    public static void main(String[] args) {
    int number = 10007*100003;
    // Check if the number is prime
    int i = 2;
    boolean isPrime = true;
    while (i <= Math.floor(Math.sqrt(number))) {
        if (number % i == 0) {
            isPrime = false;
            break;
        }
        i++;
    }
    if (isPrime) {
        System.out.println(number + " is a prime number.");
    } else {
        System.out.println(number + " is not a prime number.");
    }
}
}


